quinn-proto/src/connection/mod.rs

```diff
@@ -243,7 +243,7 @@ impl Connection {
         };
         let state = State::Handshake(state::Handshake {
             rem_cid_set: side.is_server(),
-            token: None,
+            token: Bytes::new(),
```

Comment:
Because `None` is always handled exactly as if it were the empty `Bytes`. It's not a sentinel value, because we don't detect it and thereby special-case behavior: because the normal case is literally an empty token, the empty `Bytes` produces the correct behavior without special effort.